components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt

```diff
@@ -36,6 +36,7 @@ object ErrorPages {
             .replace("%button%", context.getString(errorType.refreshButtonRes))
             .replace("%messageShort%", context.getString(errorType.titleRes))
             .replace("%messageLong%", context.getString(errorType.messageRes, uri))
+            .replace("<ul>", "<ul role=\"presentation\">")
```

Comment:
This is a good start but in some cases is not going to match as consumers could use multiple variations of `<ul>`. In addition to this, I believe we have to document the issue and indicate to consumers what is the best approach. What do you think? :)

Reply:
IMO in this case <`ul>` was used just for the bullet formatting it provides. The change impacts only the way screen readers handle the content labeling of the items. 
In other cases, like using a `<ol>` the `<li>` semantics is important to be kept ( like announcing a specific ordered). That is why I think we should not touch other use cases unless a specific request will be made.

Reply:
Sorry, I meant that the consumers  could use multiple variations of `<ul>` like `<ul class="name">` or `<ul id="id">` and `replace("<ul>", "<ul role=\"presentation\">")` won't be able to add the `role=\"presentation\"`  property as it won't  match. I just wanted to have a sometime type of documentation to indicate that issue to consumer or given them.

Reply:
I'm completely agree with you it will fix the actual issue. Feel free to land!